concourse/scripts/pipelines-cloudfoundry.sh

```diff
@@ -11,9 +11,11 @@ config_autodelete="${SCRIPT_DIR}/../pipelines/autodelete-cloudfoundry.yml"
 
 generate_vars_file() {
    set -u # Treat unset variables as an error when substituting
+   local aws_account=${AWS_ACCOUNT:-dev}
    cat <<EOF
 ---
-aws_account: ${AWS_ACCOUNT:-dev}
+aws_account: ${aws_account}
+auto_deploy: $([ "${aws_account}" != "dev" ] && echo "true" || echo "false")
```

Comment:
I was equally uncomfortable with the name keying on the AWS account, but was trying to avoid adding further configuration environment variables with defaults - they always seem to get accidentally omitted at some point leading to environments that are not configured as expected. In the case of this, we'd have environments that were silently not updating.

I'm happy to pull this back into being an env var however I'd really like to ensure that we're storing the settings used to generate our production environments somewhere - right now the only record of the environment variables used to generate an environment is the resulting changes to configuration inside the environment itself.

Perhaps this is a job for the proposed Makefile?